website/i18n/en.json

```diff
@@ -161,6 +161,15 @@
       "interactionmanager": {
         "title": "InteractionManager"
       },
+      "intro-react-native-components": {
+        "title": "Core Components and Native Components"
```

Comment:
Same comment as above, I'm not super fond of "XX Components and YYY Components" - feels redundant.

Reply:
removing base don previous comment